README.md

```diff
@@ -44,7 +44,8 @@ functionality themselves, but are required to call certain methods provided
 by the conversion traits:
 - `KnownLayout` indicates that zerocopy can reason about certain layout
   qualities of a type
-- `Immutable` indicates that a type does not contain any `UnsafeCell`s
+- `Immutable` indicates that a type is free from interrior mutation,
```

Comment:
```suggestion
- `Immutable` indicates that a type is free from interrior mutability,
```

Let's standardize on "mutability" rather than "mutation"? The PR currently uses both, but IMO "mutability" is the more standard term.

Reply:
Done